### PR TITLE
Reporter web app improvements

### PR DESCRIPTION
**Pull request by tsteenbe**
See individual commits for details.

### CODE REVIEW

File: reporter-web-app/src/reducers/selectors.js [hunk 1]
```diff
@@ -205,6 +205,7 @@ export const getTableViewProjectFilterSelections = memoizeOne(
         const webAppOrtResult = getOrtResult(state);
```
Comment (fviernau) [inline reporter-web-app/src/reducers/selectors.js:205]: commit message: 
'hte' -> 'the'
remove double space after 'drop-down'
a.text is one time in ticks, one time it is not.

File: reporter-web-app/src/models/WebAppOrtResult.js [hunk 2]
```diff
@@ -199,7 +199,7 @@ class WebAppOrtResult {
             if (obj.scopes) {
                 const { scopes } = obj;
                 for (let i = 0, len = scopes.length; i < len; i++) {
-                    const webAppScope = new WebAppScope(scopes[i]);
+                    const webAppScope = new WebAppScope(scopes[i], this);
```
Comment (fviernau) [inline reporter-web-app/src/models/WebAppOrtResult.js:171]: commit-msg:

scopes excludes -> scope excludes
'will now' -> maybe better: now shows

The bottom sentence could be made more compact, e.g.:

This change builds on top of eb01824 and c12995c.

File: reporter-web-app/src/components/PackageFindingsTable.js [hunk 3]
```diff
@@ -23,128 +23,111 @@ import { Table } from 'antd';
 import { CopyrightOutlined, FileTextOutlined } from '@ant-design/icons';
 
 // Generates the HTML to display scanFindings as a Table
```
Comment (fviernau) [inline reporter-web-app/src/components/PackageFindingsTable.js:25]: commit-msg:

previous showed package's scan results. -> previously shown package.
has now been fixed -> is now fixed

**Comment by fviernau** [comment]
Consider aligning also the line wrapping of the commit messages.

File: reporter-web-app/src/components/PackageFindingsTable.js (hunk 3 above)
Comment (tsteenbe) [reply reporter-web-app/src/components/PackageFindingsTable.js:25]: fixed

**Pull request merged**